### PR TITLE
[Config] add mtp loss

### PR DESCRIPTION
**Pull request by Waynezee**
### PR Category

User Experience

### PR Types

Others

### Description

原始行为
add_mtp_loss=false:  total_loss = lm_loss
add_mtp_loss=true:  total_loss = lm_loss + mtp_loss - mtp_loss.detach()

合入本PR后
add_mtp_loss=false:  total_loss = lm_loss + mtp_loss - mtp_loss.detach()
add_mtp_loss=true:  total_loss = lm_loss + mtp_loss
### 是否引起精度变化

是

影响所有开MTP的模型

### CODE REVIEW

**Review by From00 — APPROVED** [review]
LGTM
